src/modules/locale/Config.cpp

```diff
@@ -370,7 +370,7 @@ Config::setLCLocaleExplicitly( const QString& locale )
 QString
 Config::currentLocationStatus() const
 {
-    return tr( "Set timezone to %1/%2." )
+    return tr( "Timezone has been set to %1/%2", "@info" )
```

Comment:
Progress or label?

Reply:
This one is a bit unusual. I'll call it "info". This particular string is used in the "summary" page telling the user what is going to happen -- not what has happened. So this particular string change is wrong, it should remain "Set timezone to .." and maybe be tagged `@action` or .. um .. something-that-will-be-done.